client/src/components/ConcertCardBig.js

```diff
@@ -0,0 +1,164 @@
+import styled from "@emotion/styled";
+import React from "react";
+import colors from "../utils/colors";
+import Location from "../icons/location.png";
+import Date from "../icons/date.png";
+import Euro from "../icons/euro.png";
+import Train from "../icons/train.png";
+
+export default function ConcertCardBig() {
```

Comment:
In file client/src/components/BuddyRequestFormular.js, I can already see a component named `ConcertCardBig`. Maybe one is not needed or can be renamed?

Reply:
Oh, the function name in BuddyRequestFormular.js is wrong. Good that you saw it...